README.md

```diff
@@ -422,12 +422,11 @@ specific to the sensor you are using.
 
 ### Sensor Calibration File
 
-By default, scos-sensor will use `configs/default_calibration.json` as the sensor
-calibration file. However, if`configs/sensor_calibration.json` or
-`configs/sigan_calibration.json` exist they will be used instead of the default
-calibration file. Sensor calibration files allow scos-sensor to apply a gain based
-on a laboratory calibration of the sensor and may also contain other useful
-metadata that characterizes the sensor performance. For additional
+By default, scos-sensor will use `configs/default_sensor_calibration.json` as the sensor
+calibration file. However, if `configs/sensor_calibration.json` exists it will be used
+instead of the default calibration file. Sensor calibration files allow SCOS Sensor to
```

Comment:
Suggest rewording this slightly to avoid confusion. Change `Sensor calibration files` to `The calibration files` to clarify that this sentence refers to calibration files in general and not specifically sensor_calibration.json.

Reply:
Updated README in 5680801a6837a66ee28103b8b509b4eb42ac2602